New package: Diplodocus v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: Diplodocus
- Repository: https://github.com/cneverett/Diplodocus.jl
- Created by: @cneverett
- Version: v0.0.1
- Commit: 007b7561ffd6e1f094e4a63e55d55ab9bb4a520a
- Reviewed by: @cneverett
- Reference: https://github.com/cneverett/Diplodocus.jl/commit/007b7561ffd6e1f094e4a63e55d55ab9bb4a520a#commitcomment-164288071
- Description: Documentation for the DIPLODOCUS framework mesoscipic modelling of astrophysics sources
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd687be62298300397ee05d72a0ced58683d8c90c6141c1bb16e17d1c1a9e62158222a0686d899a7ee82de9d9a96b6a02232f9b6cdcc5d77ad31565bbb39ffc287fe4ac9bfec885be7084bcf0e74406cf4b91d5db1c90fdf22787c838ccaa132a3378c137b2141f4bd2bc82aada8fcbe58e44f30daec8c4a0e2e43d93da33e26fb43fc9a099ff46370a0e076c2f3e67b5a4db2ed0a8c32a6c55cb40dd8f38356329e830f42db5167bf5ebe6831426430933 -->

goerz commented:
As in https://github.com/JuliaRegistries/General/pull/136048#issuecomment-3170691537, this should have a bit more documentation before it is ready for registration. You seem to be well on your way with https://cneverett.github.io/Diplodocus.jl/dev/, but right now, that does not give enough detail to get started with using the package.

You also want to have a very brief overview and a link to the full documentation in the README.

cneverett commented:
That's okay. The packages are in a usable and tested state even at this version 0.0.1. If I add the instatllation information and a bit more description, will that be enough?

cneverett commented:
This isn't really a place holder package, more of a soft release

cneverett commented:
No problem. I'll get the documentation and installation guides up for this soft release soon and then attempt to re-register.

JuliaRegistrator committed:
New package: Diplodocus v0.0.1

UUID: 792d114c-f8c3-4b81-9580-cc03468a5b58
Repo: https://github.com/cneverett/Diplodocus.jl.git
Tree: 1c33245f6bbed7af41801e6937a19a3b79add53c

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
